src/Mod/PartDesign/Gui/TaskDressUpParameters.cpp

```diff
@@ -263,6 +287,10 @@ bool TaskDressUpParameters::KeyEvent(QEvent *e)
                 kevent->accept();
                 return true;
             }
+            if (selectAllEdgesAction && kevent->key() == (Qt::Key_Shift | Qt::Key_Control | Qt::Key_A)){

```

Comment:
A bit-wise OR of different enum types is deprecated with C++20. Instead create a QKeySequence object as above and compare it with the kevent pointer. Qt provides a suitable comparison operator for this.